frontend/src/components/SignIn/SignIn.js

```diff
@@ -19,8 +19,9 @@ class SignIn extends Component {
   render() {
     return (
       <div>
-        <h3>Log In</h3>
         <form className="form-signin">
+          <h3>Log In</h3>
+          <br></br>
```

Comment:
I was just looking through and saw this. You don't need to close `<br>` tags; they stand alone.